diktat-analysis.yml

```diff
@@ -146,7 +147,8 @@
   configuration: {}
 - name: FILE_WILDCARD_IMPORTS
   enabled: true
-  configuration: {}
+  configuration:
+    allowedWildcards: "" # Allowed wildcards for imports (e.g. "import org.cqfn.diktat.*, import org.jetbrains.kotlin.*")
```

Comment:
Do we need to have useRecommendedImportsOrder like in others yml config files?

Reply:
Yes, you are right, I'll add it

Reply:
And what if someone will want to sort wildcart imports and don't want to sort non-wildcard?

Reply:
I don't think there is much profit in separating wildcard and non-wildcard imports in this case. For example, IDEA doesn't separate them.

Reply:
Are you using `useRecommendedImportsOrder` from `FILE_WILDCARD_IMPORTS` anywhere?

Reply:
I already moved it to `FILE_UNORDERED_IMPORTS`, it was initially by mistake in `FILE_WILDCARD_IMPORTS `

Reply:
Ok